integration-test-common/src/test/java/org/apache/gravitino/integration/test/util/CommandExecutor.java

```diff
@@ -47,10 +47,9 @@ public static Object executeCommandLocalHost(
       Map<String, String> env) {
     List<String> subCommandsAsList = new ArrayList<>(Arrays.asList(command));
     String mergedCommand = StringUtils.join(subCommandsAsList, " ");
-
     LOG.info("Sending command \"{}\" to localhost", mergedCommand);
 
-    ProcessBuilder processBuilder = new ProcessBuilder(command);
+    ProcessBuilder processBuilder = new ProcessBuilder(subCommandsAsList);
```

Comment:
Can you please explain why changing this could avoid command injection?

Reply:
Let a example. If we the command is a string value, `ProcessBuilder` will take it as a whole command like `ls -al;  rm -rf`,  however, if we split it into a list, `-al;`, `rm`, `-rf` will be reviewed as the parameter of the first value, thus dangerous operation like `rm -rf` will not take effect.

Reply:
Can you please put it on the PR description.

Reply:
> Let a example. If we the command is a string value, `ProcessBuilder` will take it as a whole command like `ls -al; rm -rf`, however, if we split it into a list, `-al;`, `rm`, `-rf` will be reviewed as the parameter of the first value, thus dangerous operation like `rm -rf` will not take effect.

It's really hard to understand the meanings of what you say, can you please rephrase the words.

Reply:
Why changing from array to list will solve the problem?

Reply:
```
List<String> subCommandsAsList = new ArrayList<>(Arrays.asList(command));
```
Why do we need to wrap a list again with ArrayList.